src/main/java/org/folio/innreach/client/InstanceStorageClient.java

```diff
@@ -0,0 +1,14 @@
+package org.folio.innreach.client;
+
+import org.folio.innreach.config.FolioFeignClientConfig;
+import org.folio.innreach.domain.dto.folio.inventoryStorage.InstanceIterationRequest;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@FeignClient(name = "instance-storage", configuration = FolioFeignClientConfig.class)
+public interface InstanceStorageClient {
+
+  @PostMapping("/instances/iteration")
+  void startInitialContribution(@RequestBody InstanceIterationRequest request);
```

Comment:
The response shouldn't be empty. Please check the wiki [page](https://wiki.folio.org/display/FOLIJET/SPIKE%3A+%5BMODINREACH-80%5D+Record+Contribution%3A+Analyze+Re-Index+job+implementation+usage+in+mod-inventory-storage#SPIKE:[MODINREACH80]RecordContribution:AnalyzeReIndexjobimplementationusageinmodinventorystorage-Introducenew%22InstanceIteration%22APIinterface) to find the endpoint definition